addons/docs/package.json

```diff
@@ -40,6 +40,7 @@
     "common/**/*",
     "ember/**/*",
     "html/**/*",
+    "svelte/**/*",
```

Comment:
So the svelte app, has a `XYZ.svelte` decorator, which we do not parse/compile/transpile at all. To us, it's NOT source.

before this file was copied over (multiple times) to all the `dist` folders (duplicated).
besides being inefficient, and increasing the size of our packages (very minuscule), we can just point to a file relative to the package root, because why have babel copy the file 3 times if it can just exist in 1 place instead?